llama_stack/providers/remote/vector_io/milvus/milvus.py

```diff
@@ -101,7 +136,17 @@ async def query_keyword(
         k: int,
         score_threshold: float,
     ) -> QueryChunksResponse:
-        raise NotImplementedError("Keyword search is not supported in Milvus")
+        # Simple text search using content field
+        search_res = await asyncio.to_thread(
+            self.client.query,
+            collection_name=self.collection_name,
+            filter=f'content like "%{query_string}%"',
```

Comment:
looking at this, it looks like rather than using the Full-Text search (i.e., sparse search using BM25 as mentioned in the documentation [you originally linked](https://milvus.io/docs/full_text_search_with_milvus.md#Perform-Retrieval)), it looks like this is doing naive exact match. It's worth mentioning this is very sensitive to small changes. I'd recommend considering the the Full-Text search approach, which allows you to specify lexical analysis parameters (e.g., stemming, stop word removal, and lower casing).  See the [analyzer parameters here](https://milvus.io/docs/analyzer-overview.md).